cmd/okctl/applycluster.go

```diff
@@ -125,6 +126,11 @@ func buildApplyClusterCommand(o *okctl.Okctl) *cobra.Command {
 			reconciliationManager.AddReconciler(resourcetree.ResourceNodeTypeExternalDNS, reconciler.NewExternalDNSReconciler(services.ExternalDNS))
 			reconciliationManager.AddReconciler(resourcetree.ResourceNodeTypeGithub, reconciler.NewGithubReconciler(services.Github))
 			reconciliationManager.AddReconciler(resourcetree.ResourceNodeTypeIdentityManager, reconciler.NewIdentityManagerReconciler(services.IdentityManager))
+			reconciliationManager.AddReconciler(
```

Comment:
Are we not going to do this whole ns dance for okctl create cluster?

Reply:
`apply cluster` will likely superceed `create cluster`. Are you suggesting to implement it in `create cluster` anyway? I could do it, but not in this PR

Reply:
Ok, I didn't know it was going to replace it, but in that case it's okay.

Reply:
It hasn't been decided, I'm just assuming